apps/website/screens/components/dialog/overview/DialogOverviewPage.tsx

```diff
@@ -0,0 +1,149 @@
+import { DxcBulletedList, DxcFlex, DxcParagraph } from "@dxc-technology/halstack-react";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import Image from "@/common/Image";
+import content from "./images/dialog_content.png";
+import overlay from "./images/dialog_overlay.png";
+import anatomy from "./images/dialog_anatomy.png";
+
+const sections = [
+  {
+    title: "Introduction",
+    content: (
+      <DxcParagraph>
+        The dialog component is a modal window that captures user attention for critical interactions, such as
+        confirmations, alerts, or form inputs. It appears above the main content and requires user action before
+        continuing. To maintain usability, dialogs should be concise, focused on a single task, and provide clear
+        actions like confirmation or dismissal.
+      </DxcParagraph>
+    ),
+  },
+  {
+    title: "Anatomy",
```

Comment:
The anatomy section does not match Lara's doc:

![image](https://github.com/user-attachments/assets/112b0431-6a16-4711-826a-4da4ca81153b)